CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- BREAKING CHANGE: Support metadata field `include_info_sections` ([bug 1866559](https://bugzilla.mozilla.org/show_bug.cgi?id=1866559))
```

Comment:
Breaking in the " Any breaking changes to language binding APIs are noted explicitly " sense? Rust and Swift codegen shifted.

But I suppose those could be "internal" API changes, and I'm just not used to thinking in those terms

Reply:
@travis79 So, uh... is this breaking or not? Should I land as-is and cut a major version release, or is this internal (because it's below the language bindings layer) and I should cut a minor version release?

Reply:
Ahh, thanks @badboy, that makes sense.